qiskit_nature/second_q/transformers/active_space_transformer.py

```diff
@@ -91,36 +85,26 @@ class ActiveSpaceTransformer(BaseTransformer):
 
     def __init__(
         self,
-        num_electrons: Optional[Union[int, Tuple[int, int]]] = None,
-        num_molecular_orbitals: Optional[int] = None,
-        active_orbitals: Optional[List[int]] = None,
+        num_electrons: int | tuple[int, int],
+        num_spatial_orbitals: int,
```

Comment:
I guess I have to ask - in the past we have molecular and spin. Is this the same as the first? Or maybe this is more ambiguous to cope with AO as well as MO?

Reply:
Yes, this is ambiguous w.r.t. AO/MO/NAO/whatever one comes up with.
One spatial orbital is then split into two spin orbitals for the up- and down-spin, respectively.